src/scipp/testing/strategies.py

```diff
@@ -0,0 +1,225 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+# @author Jan-Lukas Wynen
+from functools import partial
+from typing import Optional, Sequence, Union
+
+from hypothesis.errors import InvalidArgument
+from hypothesis import strategies as st
+from hypothesis.extra import numpy as npst
+
+from ..core import variable as creation
+from ..core import DataArray, DType
+
+# TODO sometimes checks fail with (seen in test_astype_int_to_float)
+#  hypothesis.errors.FailedHealthCheck: Examples routinely exceeded the max
+#  allowable size. (20 examples overran while generating 9 valid ones).
```

Comment:
Until is fixed, we should keep these tests out of CI, and make sure that, e.g., running `pytest` in the `tests` folder does not run them?

Reply:
Should be fixed now. I have not encountered this problem with the new implementation. See new comment above `variables`. I tried to break it by generating a lot of examples and giving it hard errors to find and produce an MWE.